README.md

```diff
@@ -29,6 +34,13 @@ kubectl apply -f https://github.com/kyma-project/docker-registry/releases/latest
 kubectl apply -f https://github.com/kyma-project/docker-registry/releases/latest/download/default-dockerregistry-cr.yaml
```

Comment:
Please indent the code blocks in this section to make the numbered points better visible.